index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Ishpaul Singh</title>
+</head>
+<body>
+    <header>
+        
+        <nav>
+            
+            <div class="logo">
+                <h1>ISHPAUL SINGH</h1>
+            </div>
+            
+            <div class="toggle-menu">
+                <a href="#"><img src="assets/icons/menu-icon.svg" alt="menu"></a>
+            </div>
+        </nav>
+
+    </header>
+        
+        <section class="intro-section">
+            
+            <p class="headline">
+                Hii!<br>
+                I'm Ishpaul Singh<br>
+                Glad to see you!
+            </p>
```

Comment:
- [ ] Kindly make sure to stick as much as possible to the Figma design, which includes:
   - [ ] Adjusting the spacing on the right side of this section to match the Figma design.
   - [ ] Using the same font style as in the Figma design which is `Poppins` you can get it from [Google fonts](https://fonts.google.com/).

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/170314309-4ba7f8f8-1f74-48c4-aa18-8a2dcf7b7132.png) | ![image](https://user-images.githubusercontent.com/62290545/170314420-8dbf3006-4360-4a2c-b162-20f367b00995.png) |